src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFliteTagMercury.java

```diff
@@ -0,0 +1,36 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.templates.mobile.MobileTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+
+import org.testng.annotations.Test;
+
+/**
+ * @ownership: AdEngineering Wikia
```

Comment:
Yep, I am +1 for `AdEngineering Wikia` as well.
However, for me more important is to keep consistency for all adeng tests rather than difference between `AdEng` and `AdEngineering Wikia`.